zerver/openapi/zulip.yaml

```diff
@@ -4025,6 +4112,14 @@ components:
           type: string
           description: |
             The time zone of the user.
+        date_joined:
+          type: string
+          description: |
+            The time the user account was created.
+        profile_data:
+          type: object
+          description: |
+            Object containing profile data of the user.
```

Comment:
This can definitely be for a later PR, but we should plan to define the schema for these as a common component.

Reply:
Yes, Although I am not sure but if profile data is the same as data returned by `get_users` then we can directly use $ref as we already have a shared user schema.

Reply:
Yes, it is the same.